src/errors/NotImplementedError.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * NotImplementedError is thrown when an abstract method has not been implemented.
+ * This is a workaround to the fact that ES6 does not formally support abstract classes.
+ */
+class NotImplementedError extends Error {
+	/**
+	 * @param {String} methodName The name of the method that was not implemented.
```

Comment:
Insane quibble: do we end param descriptions with a period or not? (In the others we don't.)